frontend/src/components/common/SystemLog/TSystemLog.vue

```diff
@@ -3,9 +3,11 @@
     <div class="logs__icon-wrapper">
       <t-icon v-show="isError" class="logs__icon" icon="warning" />
     </div>
-    <div class="logs__date">{{ date }}</div>
-    <div class="logs__time">{{ time }}</div>
-    <div class="logs__info">{{ info }}</div>
+    <div class="logs__message">
```

Comment:
I have updated this component during rebase and merge.
It wasn't 100% per UI spec. Log message field was aligned to center, while it should have be aligned to the start of the element.

I also simplified style a bit.
If you need to have gaps between elements in the flex please use `gap-N` class or `gap: Npx` style instead of margins.